templates/poster.html

```diff
@@ -1,4 +1,22 @@
 {% extends "base.html" %}
+
+{% block head %}
+
+{{ super() }}
+<meta name="citation_title" content="{{paper.content.title}}" />
+{% for author in paper.content.authors %}
+<meta name="citation_author" content="{{author}}" />
+{% endfor %}
+<meta name="citation_publication_date" content="{{config.data}}" />
```

Comment:
typo “data” -> “date”

Reply:
Forgot to flag - "data" is the key used in config.yml, which is the root of this typo

Reply:
ah. thanks! #43 will fix it. Can you change it here after that PR is merged?